chapters/4-acquisition.tex

```diff
@@ -508,12 +530,180 @@ \subsection{Using electronic survey features}
+	see the DIME Wiki: \url{https://dimewiki.worldbank.org/High\_Frequency\_Checks}}
+HFCs should include monitor consistency and range of responses to each question, 
+survey programming validation, tests for enumerator-specific effects, 
+and checks for duplicate entries and completeness of online submissions vis-a-vis the field log.
+
+High-frequency checks will only improve data quality
+if the issues they catch are communicated to the data provider
+and corrections are documented and applied to the data.
+This requires close communication with the field team, 
+so that enumerators are made aware of data quality issues promptly,
+and a transparent system for document issues and corrections. 
+There are many ways to communicate results of high-frequency checks to the field team.
+What's most important is to find a way to create actionable information for your team.
+\texttt{ipacheck}, for example, generates a spreadsheet with flagged errors;
+these can be sent directly to the data collection teams.
+Many teams choose other formats to display results,
+such as online dashboards created by custom scripts.
+It is also possible to automate communication of errors to the field team
+by adding scripts to link the HFCs with a messaging platform.
+Any of these solutions are possible:
+what works best for your team will depend on such factors as
+cellular networks in field work areas, whether field supervisors have access to laptops,
+internet speed, and coding skills of the team preparing the HFC workflows.
+
+Careful \textbf{field validation} is essential for high-quality survey data.
+While we cannot control natural measurement error,\sidenote{
+	See \citet{kondylis2015measuring} for an example.}
+which comes from variation in the realization of key outcomes,
+there is often an opportunity to reduce error arising from inaccuracies in the data generation process.
+\textbf{Back-checks}, spotchecks and other validation audits help ensure that 
+data is not falsified, incomplete, or otherwise suspect.\sidenote{For instructions
+	on how to design and implement Back Checks, see the DIME Wiki: \url{https://dimewiki.worldbank.org/Back\_Checks}}
+Field validation is also an opportunity to ensure that all field protocols are followed.
+For back-checks, a random subset of observations is selected,
+and a subset of information from the full survey is
+verified through a brief targeted survey with the original respondent.
+For spot-checks, field supervisors (and, if contextually appropriate, research team staff)
+should perform unannounced accompaniment of all enumerators, 
+to confirm first-hand that survey protocols are followed and 
+survey questions are well understood. 
+A complementary approach is audio audits, 
+in which a subset of the interview is recorded, and listened to by a trained data validator
+(audio audits require additional permissions, and must be included in informed consent).
```

Comment:
Seems like we start to discuss audio audits here, then go back to back-checks, and then the section ends on audio audits. Is there something I do not understand about the workflow that makes us go back and forth?

Reply:
removed redundant sentence on audio audits b4e56df23efcc633b500791abbd7d7a9e08fe886